src/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -36,6 +36,8 @@ namespace Microsoft.Identity.Client.AppConfig
 {
     internal sealed class ApplicationConfiguration : IApplicationConfiguration
     {
+        public bool UseCorporateNetwork { get; internal set; }
+
```

Comment:
does iOSKeychainSecurityGroup need to be here too?

Reply:
Maybe only in PublicClientApplicationConfiguration?

Reply:
Yes, we should add WithIosKeychainSecurityGroup, will do...


---
In reply to: [250856931](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/833#discussion_r250856931) [](ancestors = 250856931)

Reply:
that doesnt exist.  the config is clientapp wide and internal facing.

---
In reply to: [250968430](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/833#discussion_r250968430) [](ancestors = 250968430)